app/src/ui/diff/syntax-highlighting/index.ts

```diff
@@ -120,7 +120,11 @@ export async function getFileContents(
     }),
   ])
 
-  return { file, oldContents, newContents }
+  return {
+    file,
+    oldContents: oldContents.toString('utf8'),
+    newContents: newContents.toString('utf8'),
```

Comment:
This conversion from `Buffer` to `utf8` string was done anyway, I just moved it here for convenience.